web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -340,7 +340,7 @@ import {get_string_diff} from "../../src/util";
         },
 
         matcher(item) {
-            return ~item.toLowerCase().indexOf(this.query.toLowerCase());
+            return item.toLowerCase().includes(this.query.toLowerCase());
```

Comment:
Is that previous `~` supposed to have been https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_NOT? 

I guess that may be often equivalent to whether it's -1, but bleh.

Reply:
You’re complaining about a correct rewriting because the old version temporarily confused you? Seems like a great reason to rewrite it!

Reply:
I learned about this here https://wsvincent.com/javascript-tilde/

Reply:
I'm not complaining, just being dragged into remembering the bad old days of JS development.